draft-ietf-scitt-architecture.md

```diff
@@ -165,29 +170,34 @@ Issuers can register their Signed Statements on one or more Transparency Service
 
 # Introduction
 
-This document describes the generic, interoperable, and scalable SCITT architecture.
-Its goal is to enhance auditability and accountability across supply chains.
-
-In supply chains, downstream Artifacts are built upon upstream Artifacts.
-The complexity of traceability and quality control for these supply chains increases with the number of Artifacts and parties contributing to them.
-There are many parties who publish information about Artifacts:
-For example, the original manufacturer may provide information about the state of the Artifact when it left the factory.
-The shipping company may add information about the transport environment of the Artifact.
-Compliance Auditors may provide information about their compliance assessment of the Artifact.
-Security companies may publish vulnerability information about an Artifact.
-The original manufacturer may subsequently provide additional information about the manufacturing process they discovered after the Artifact left the factory.
-Some of these parties may publish information about their analysis or use of an Artifact.
-
-SCITT provides a way for Relying Parties to obtain this information in a way that is "transparent", that is, parties cannot lie about the information that they publish without it being detected.
-SCITT achieves this by having producers publish information in a Transparency Service, where Relying Parties can check the information.
+This document defines an architecture, a base set of extensible message structures and associated flows to make signed data items transparent via verifiable data structures maintained by corresponding transparency services.
+The goal of the transparency enabled by the Supply Chain Integrity, Transparency, and Trust (SCITT) architecture is to enhance auditability and accountability for signed data items (statements) that are about supply chain commodities (artifacts).
+Registering signed statements with a transparency service is akin to a notarization procedure.
+Transparency services therefore act like notaries that each record (register) a single, linear and irrevocable history of statements that were made.
```

Comment:
```suggestion
Transparency services perform notary operations, confirming a policy is met before recording the statement on the ledger.
One the signed statement is registered, the notary issues a receipt, signed by the transparency service, just as a notary stamps the document being notarized.```

Reply:
Do we want ledger here this early? Single, linear and irrevocable history seemed more descriptive and introductory. Agree it's good to mention the policy check though.

Reply:
It's a good question on balance. In this case, I was using Ledger, as it applies to a notary, as it's a mappable concept. I didn't directly mean to remove "linear and irrevocable history of statements that were made.", rather I couldn't find a good way to stitch it in. 
Please look at the updated suggestion.

Reply:
New version looks good to me!